src/components/Navigator/NavigatorCard.vue

```diff
@@ -524,6 +547,7 @@ export default {
       const technology = sessionStorage.get(STORAGE_KEYS.technology);
       const nodesToRender = sessionStorage.get(STORAGE_KEYS.nodesToRender, []);
       const filter = sessionStorage.get(STORAGE_KEYS.filter, '');
+      const apiChanges = sessionStorage.get(STORAGE_KEYS.apiChanges);
```

Comment:
The boolean is to cast the API changes prop to boolean. Basically, if we had API changes enabled and still do, reuse the persisted items, otherwise just regenerate the items to render.

I have changed the property as you suggested.